src/index.ts

```diff
@@ -478,6 +483,13 @@ function enhanceDB(db: InternalDB, options: DBParams): DB {
 
       return db.executeRaw(query, sanitizedParams as Scalar[]);
     },
+    executeRawSync: (query: string, params?: Scalar[]) => {
+      const sanitizedParams = sanitizeArrayBuffersInArray(params);
+      let res = sanitizedParams
+        ? db.executeSync(query, sanitizedParams as Scalar[])
```

Comment:
this doesn't look correct. If you take a look at the function above a executeRaw is a different native implementation than executeSync.